quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/HelpCommandDistTest.java

```diff
@@ -165,10 +165,19 @@ private void assertSingleJvmStarted(CLIResult run) {
     }
 
     private void assertHelp(CLIResult result) {
```

Comment:
@vmuzikar As you can see, the logic in `HelpCommanDistTest` test varies for the product and community as the GELF is not mentioned in the help output. I'm not sure if it's worth the effort to maintain even the version without GELF for help command. 

I'd say probably not, and perhaps ignoring the test for the product would be preferred. 

@vmuzikar WDYT?